Automattic/jetpack
# Actions: Test #18558

I'll start by not adding any labels to that PR, and I'll keep that description too short.

**Edit**: here is what the comment below looked like:

![image](https://user-images.githubusercontent.com/426388/107764871-30885c00-6d31-11eb-9f63-d2ef9ae6ac94.png)

Looks like I should have kept it even shorter. :)

Now I'll add a Status label.

**Edit**: here is what the comment looked like:

![image](https://user-images.githubusercontent.com/426388/107765070-7d6c3280-6d31-11eb-8029-fa5864e58305.png)

Now I'll add testing instructions:

#### Testing instructions:

* Check the comments on this PR.

**Edit**: here is what the comment looked like (you could also check the revision history of the comment):

![image](https://user-images.githubusercontent.com/426388/107765211-b5737580-6d31-11eb-8431-fb1d099addac.png)

Now I'll add a changelog entry and a privacy section.

#### Does this pull request change what data or activity we track or use?

* No

#### Proposed changelog entry for your changes:

* N/A

**Edit**: here is what the comment looked like:

![image](https://user-images.githubusercontent.com/426388/107765335-eb185e80-6d31-11eb-8e19-d435f1d622c8.png)

Now I'll rebase this PR to remove the non-verified commit.

**Edit**: here is what the comment looked like:

![image](https://user-images.githubusercontent.com/426388/107765621-78f44980-6d32-11eb-9167-84ae3ee76eee.png)

**********

After the last round of updates, the milestone info is only added if a plugin label has been added before. The comment consequently looks like this by default:

![image](https://user-images.githubusercontent.com/426388/107791355-20ce3f00-6d54-11eb-9af6-39c1b2170368.png)

**Edit** After adding both the Jetpack plugin and the Beta plugin labels, info about both plugins is added to the bottom of the comment:

![image](https://user-images.githubusercontent.com/426388/107791526-5541fb00-6d54-11eb-9d1f-c276657f6fb4.png)

*******

Since one of the tests is failing, the Needs Author reply is added:

![image](https://user-images.githubusercontent.com/426388/107796866-9b01c200-6d5a-11eb-84dc-c80497247c6a.png)


********

Since one of the tests is failing, the Needs review label is removed, and the Needs Author reply is aded instead:

![image](https://user-images.githubusercontent.com/426388/107797028-c4225280-6d5a-11eb-832d-46d4d541b501.png)
